bundles/CoreBundle/Resources/config/authenticator_security.yaml

```
@@ -0,0 +1,6 @@
+# Pimcore
+pimcore:
+    security:
+        factory_type: password_hasher
+        password_hasher_factories:
```

```suggestion
        encoder_factories:
```
Should we use the new name here already?

---

yes, this config is only [loaded ](https://github.com/pimcore/pimcore/pull/10967/files/7ca5aaec0306cf17510daebb5fd2420fde57f888#diff-21e7a0c46b9617b0a01d857b82e976093d753270454737dcf2ace8e171e8a7e9R41)when you enable new authenticator otherwise will keep using the old factory name [here](https://github.com/pimcore/pimcore/blob/10.x/bundles/CoreBundle/Resources/config/pimcore/default.yaml#L102)